pkg/products/threescale/filters.go

```diff
@@ -0,0 +1,61 @@
+package threescale
+
+import (
+	"github.com/openshift/api/route/v1"
+	"strings"
+)
+
+type TenantAccountsFilter struct {
+	providers  []AccountDetail
+	developers []developerAccounts
+}
+
+type developerAccounts struct {
```

Comment:
As we discussed earlier, this struct's name should probably be singular. To more accurately represent its data it should be called `developerRoute` or something similar